src/products/executable_generators.jl

```diff
@@ -58,10 +58,11 @@ macro declare_executable_product(product_name)
 end
 
 macro init_executable_product(product_name, product_path)
+    preference_name = string(product_name, "_path")
     path_name = Symbol(string(product_name, "_path"))
```

Comment:
Maybe this can be
```suggestion
    path_name = Symbol(preference_name)
```
to ensure consistency?

Reply:
I thought about that; but I'm not entirely sure I want the preference name to be `libfoo_path`; perhaps just `libfoo` is better?  What do you think?

Reply:
I see, good point.  For the preference the `_path` is probably redundant